admin/controllers/service-manager.js

```diff
@@ -139,15 +139,15 @@ const getCheckWindows = async (req, res, next) => {
 
   // Get current check windows
   try {
-    checkWindows = await checkWindowDataService.fetchCurrentCheckWindows(sortField, sortDirection)
-    checkWindowsCurrent = checkWindowService.formatCheckWindowDocuments(checkWindows, true)
+    checkWindows = await checkWindowDataService.sqlFetchCurrentCheckWindows(sortField, sortDirection)
+    checkWindowsCurrent = checkWindowService.formatCheckWindowDocuments(checkWindows, true, false)
   } catch (error) {
     return next(error)
   }
 
   // Get past check windows
   try {
-    checkWindows = await checkWindowDataService.fetchPastCheckWindows(sortField, sortDirection)
+    checkWindows = await checkWindowDataService.sqlFetchPastCheckWindows(sortField, sortDirection)
```

Comment:
s/Fetch/Find/ ?  Again CheckWindow is not really needed in the function name, although some may prefer it.